### ISSUE TITLE
Reprinting "return 1, 2, 3;" outputs "return (1, 2, 3);"

### ISSUE THREAD

**Issue by spicyj** [benjamn/recast#261]

**Comment by benjamn** [comment]
Yeah, the printer is a bit aggressive about parenthesizing `SequenceExpression`s. I realize it's not ideal, but are you facing any correctness problems from this quirk? Otherwise a good minifier should handle it?

**Comment by spicyj** [comment]
Just codemods that change lines that don't need to be changed. @avikchaudhuri ran into it.
